### DISCUSSION TITLE
celery task_runner does not start

### DISCUSSION THREAD

**Discussion by Silver-Sky** [codingjoe/joeflow#113] [Q&A]
Hello Joe

I tried your application, and after I was able to run the tutorial, it seems that the machine tasks were not started with celery at all.

I solved the problem by removing some attributes from the task_runner and adding the runner to my tasks.py file:
```
import logging

from joeflow.runner.celery import _celery_task_runner

logger = logging.getLogger(__name__)


def task_runner(*, task_pk, workflow_pk, countdown, eta):
    """Schedule asynchronous machine task using celery."""
    _celery_task_runner.apply_async(
        args=(task_pk, workflow_pk),
        # countdown=countdown,
        # eta=eta,
        # queue=settings.JOEFLOW_CELERY_QUEUE_NAME,
    )
```

and register my runner in the settings: JOEFLOW_TASK_RUNNER = "workflow.tasks.task_runner"

That way it works like a charm. I used celery==5.3.6.

Maybe that is not an Issue of your code, but I just want you (and other users) to know in case they have the same issue.

**Comment by codingjoe** [comment]
Hi @Silver-Sky,

Thanks for reaching out. Are your workers listening to the `jowflow` queue?
Maybe double-check your Celery setup and carefully read through our documentation.

If you are stuck, please don't hesitate to reach out again.

Cheers!
Joe
